carbon-design-system/carbon
# fix(FileUploaderDropContainer): pass through empty `accept` attr

Closes #4519

Previously if the user did not supply any file type restrictions, the resulting validated files array would be empty. This PR adds a check for empty `accept` attribute on drop events to pass through drag and dropped files when the `accept` attribute is not specified

This PR also adds a code sandbox example application using the dnd uploader

#### Changelog

**New**

- Code Sandbox drag and drop file uploader example

**Changed**

- `accept` prop check in `validateFiles` method
- move `max-width` rule to drop container label to fix a visual bug where the focus outline extended past the drop area

#### Testing / Reviewing

* Remove the default `accept` knob value in storybook
* ensure the component functions as expected
